docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,83 @@
+
+## Transaction Dependencies
+
+The `ValidTransaction` [struct](https://substrate.dev/rustdocs/master/sp_runtime/transaction_validity/struct.ValidTransaction.html) defines the `requires` and `provides` parameters to build a dependency graph of transactions. Together with `priority` (discussed below), this dependency graph allows the pool to produce a valid linear ordering of transactions.
+
+For runtimes built with FRAME, the nodes order transactions with an account-based system. Every signed transaction needs to contain a nonce, which is incremented by 1 every time a new transaction is made. For example, the first transaction from a new account will have `nonce = 0` and the second transaction will have `nonce = 1`.
+
+At minimum, FRAME-transactions have a `provide` tag of `encode(sender ++ nonce)` and a `require` tag of `encode(sender ++ (nonce -1)) if nonce > 1`. Transactions do not require anything if `nonce=0`. As a result, all transactions coming from a single sender will form a sequence in which they should be included.
+
+Substrate supports multiple `provide` and `require` tags, so custom runtimes can create alternate dependency (ordering) schemes.
+
+## Transaction Priority
+
+Transaction `priority` in the `ValidTransaction` [struct](https://substrate.dev/rustdocs/master/sp_runtime/transaction_validity/struct.ValidTransaction.html) determines the ordering of transactions that are in the ready queue. If a node is the next block author, it will order transactions from high to low priority in the next block until it reaches the weight or length limit of the block.
+
+`priority` defines the linear ordering of a graph in the case of one transaction unlocking multiple dependent transactions. For example, if we have two (or more) transactions that have their dependencies satisfied, then we use `priority` to choose the order for them.
+
+
+For runtimes built with FRAME, `priority` is defined as the `fee` that the transaction is going to pay. For example:
+
+  * If we receive 2 transactions from *different* senders (with `nonce=0`), we use `priority` to determine which transaction is more important and included in the block first.
+  * If we receive 2 transactions from the *same* sender with an identical `nonce`, only one transaction can be included on-chain. We use `priority` to choose the transaction with a higher `fee` to store in the transaction pool.
+
+Note that the pool does not know about fees, accounts, or signatures - it only deals with the validity of the transaction and the abstract notion of the `priority`, `requires`, and `provides` parameters. All other details are defined by the runtime via the `validate transaction` implementation.
+
+## Transaction Lifecycle
+
+A transaction can follow two paths:
+
+### Block Produced by our Node
+
+  1) Our node listens for transactions on the network.
```

Comment:
This is a tiny nit, but I think most lists use a period after each number and no empty line between items. It may not matter on docusaurus (some markdown renderers will end up with the same result) but if we are going to be switching platforms, then we should try to maintain a common format. So,
```
1. Words
2. Werds
3. Wurds
```